Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -36,6 +36,8 @@ def linkage
             libgcc_s.so.1
             libgomp.so.1
             libstdc++.so.6
+            
```

Comment:
Oops, there's some trailing whitespace here. Can't fix right now, will try to fix tonight if possible.